eng/build.ps1

```diff
@@ -366,14 +383,23 @@ function TestUsingRunTests() {
   if ($testCoreClr) {
     $args += " --tfm net5.0"
     $args += " --tfm netcoreapp3.1"
-    $args += " --include '\.UnitTests'"
     $args += " --timeout 90"
+    if ($testCompilerOnly) {
+      $args += GetCompilerTestAssembliesIncludePaths
+    } else {
+      $args += " --include '\.UnitTests'"
```

Comment:
```suggestion
      $args += " --include '\.UnitTests$'"
```

Consistency with the other regex in this change.

Reply:
I haven't gone through all the test projects in Roslyn to ensure that they end in `UnitTests`, and the existing regex used this format. I'd prefer to keep it as is.